spring-integration-rsocket/src/main/java/org/springframework/integration/rsocket/RSocketConnectedEvent.java

```diff
@@ -61,4 +63,13 @@ public RSocketRequester getRequester() {
 		return this.requester;
 	}
 
+	@Override
+	public String toString() {
+		return "RSocketConnectedEvent{" +
+				"destination='" + this.destination + '\'' +
+				", data=" + Arrays.toString(this.data.asByteBuffer().array()) +
```

Comment:
I don't think rendering a `byte[]` as a bunch of ints is useful, maybe omit this (or just include the length, perhaps).